packages/react-storage/src/components/StorageBrowser/context/elements/DropZoneElement.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ViewElement, ViewElementProps } from './definitions';
+import { useDropZone } from '@aws-amplify/ui-react-core';
+import { CLASS_BASE } from '../../views/constants';
+
+export interface DropZoneElementProps extends ViewElementProps {
+  handleDroppedFiles: (files: File[]) => void;
+}
+
+export const DropZoneElement = ({
```

Comment:
This doesn't match the how the other elements function. Also not sure if we need a component this component at all since the `Table` just needs the drag active conditional styling applied?